src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```

Comment:
Why do you need this? You give a type that you 100% will have a user
![image](https://user-images.githubusercontent.com/91411692/233936252-4aefb9e0-e139-4e03-b7b5-da483dfc2de5.png)
![image](https://user-images.githubusercontent.com/91411692/233936304-aa70149b-f30d-4b7e-87d5-a899feb3d8a0.png)

```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
```